File: c/bee1002.c
#include <stdio.h>
#include <math.h>
 
int main() {
    double raio, pi, area;
    pi = 3.14159;
    scanf("%lf", &raio);
    area = pow(raio, 2) * pi;

    printf("A=%.4lf", area);
    return 0;
}

File: java/bee1002.java
import java.util.Scanner;

public class bee1002 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double raio = input.nextDouble();
        double pi = 3.14159;

        double area = raio*raio*pi;

        System.out.printf("A=%.4f\n", area);
        input.close();
    }
}
